upstream.repos

```diff
@@ -14,8 +14,8 @@ repositories:
     version: ros2
   ur_msgs:
     type: git
-    url: https://github.com/destogl/ur_msgs.git
-    version: ros2
+    url: https://github.com/ros-industrial/ur_msgs.git
```

Comment:
Works for me although I think we can remove a bunch of these dependencies. I only needed to build from source:

```
moveit_task_constructor (humble)
moveit_visual_tools
rosparam_shortcuts 
rviz_visual_tools
```
Maybe consider removing the rest to greatly simplify the whole install setup